### PR TITLE
Buy Team 52135: Added option to disable auto focus on Accordion component.

### PR DESCRIPTION
**Pull request by hahn208**

### CODE REVIEW

**Review by CarlosRangel17 — CHANGES_REQUESTED** [review]
Can we make this update to our ADO component library to keep things synced up there?

**Comment by CarlosRangel17** [comment]
> Can we make this update to our ADO component library to keep things synced up there?

Also, let's be sure to update the package.json version, and update the change log file

**Review by hahn208 — COMMENTED** [review]
Updated the Changelog and Package. Also added changes that I pushed to the ADO MFCL.

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "mfcl",
-  "version": "4.0.1",
+  "version": "4.0.2",
```
Comment (CarlosRangel17) [inline package.json:3]: This should be 4.0.3 :)

File: src/Stepper/Step.tsx [hunk 2]
```diff
@@ -1,4 +1,4 @@
-import React, { FunctionComponent, ReactNode, memo, KeyboardEvent } from 'react'
+import React, {FunctionComponent, ReactNode, memo, KeyboardEvent} from 'react'
```
Comment (CarlosRangel17) [inline src/Stepper/Step.tsx:1]: These formatting changes seem unnecessary... I think the convention for this project would be to keep these spaces. We might need to align our VS Code formatter settings for this project? Yet another tech debt item on my to do list. 

Sorry to be pedantic, but can we revert these formatting changes for now?

**Review by CarlosRangel17 — APPROVED** [review]

**Comment by hahn208** [comment]
Closing to reopen and hopefully Vercel will just work.

**Pull request merged**